runtime/compiler/p/codegen/J9PPCSnippet.cpp

```diff
@@ -1364,13 +1364,13 @@ TR::PPCNonZeroAllocPrefetchSnippet::PPCNonZeroAllocPrefetchSnippet(
 uint8_t *TR::PPCNonZeroAllocPrefetchSnippet::emitSnippetBody()
    {
    TR::Compilation *comp = cg()->comp();
+   if (comp->getOptions()->realTimeGC())
```

Comment:
Same here: You moved the return statement up. Is it possible that the code executed before this return needed to stay because it had important side effects?